dashboard/app/views/levels/editors/fields/_blockly.html.haml

```diff
@@ -7,31 +7,31 @@
 
 %legend.control-legend.levelTypeHeader
   Blockly Options
```

Comment:
this file kind of falls in no mans land now, because we have partials for specific level types (which we left in levels/editors) and we have partials for individual fields, which was my thought for what should go in levels/editors/fields. My vote would be to move everything that's not an individual field up into levels/editors, but I don't feel super strongly about it so if what you have here makes more sense to you then that's fine with me too.

Reply:
I can see what you are saying. My goal is over time to make it so level/editors is only for files of level types and then for each of those level types, we have groups of fields we can add to them.  The tricky part of this though is that many levels are a combo of things (such as blockly, maze, studio) so I think it will take a little more time to get there.